tests/ClientTest.php

```diff
@@ -345,11 +346,12 @@ public function testAddConnectionWithSslTransportAndForceIpv4ThrowsException()
 
         try {
             $this->client->addConnection($connection);
-            $this->fail('Expected exception was not thrown');
-        } catch (Exception $e) {
             $this->assertSame(Exception::ERR_CONNECTION_STATE_UNSUPPORTED, $e->getCode());
+        } catch (Exception $e) {
+            $this->fail('Unexpected exception was thrown');
         }
     }
+*/
```

Comment:
This test should be modified to ensure that both of these options can now be enabled simultaneously.